Jenkinsfile

```diff
@@ -2,9 +2,7 @@
 
 pipeline {
    parameters {
-         booleanParam(defaultValue: true, description: 'Execute pipeline?', name: 'shouldBuild')
```

Comment:
These two lines were not working. Their purpose was to be used in the PR description, which is still handled below.